src/react-imgix-bg.jsx

```diff
@@ -87,14 +86,13 @@ const BackgroundImpl = (props) => {
 
   const renderedSrc = (() => {
     const [rawSrc, params] = extractQueryParams(src);
+    const allParams = { ...params, ...imgixParams, dpr };
```

Comment:
👍

Reply:
I can definitely employ more object spread to shorten some other bits of code up here too.